specification/cognitiveservices/data-plane/LUIS/Programmatic/stable/v2.0/LUIS-Programmatic.json

```diff
@@ -2,7 +2,7 @@
   "swagger": "2.0",
   "info": {
     "title": "LUIS Programmatic API",
-    "version": "v2.0"
+    "version": "v2.0 preview"
```

Comment:
Why is that?

Reply:
This package have never been published. When we publish the first version of a package, we would like to set it as preview first (like other Cognitive packages at https://www.nuget.org/packages?q=Microsoft.Azure.CognitiveServices. Then we will remove preview after it is stable later on.